Makefile

```diff
@@ -87,6 +87,9 @@ docs: build
 	./bin/process-compose docs ${DOCS_DIR}
 	for f in ${DOCS_DIR}/*.md ; do sed -i 's/${USER}/<user>/g; s|${TMPDIR}|/tmp/|g; s/process-compose-[0-9]\+.sock/process-compose-<pid>.sock/g' $$f ; done
 
+schema:
+	./bin/process-compose schema ./schema.json
```

Comment:
Can we save it under `schemas` (new dir) and name it `process-compose-schema.json`?